packages/web3torrent/src/library/web3torrent-lib.ts

```diff
@@ -148,52 +162,46 @@ export default class WebTorrentPaidStreamingClient extends WebTorrent {
     }
   }
 
+  /**
+   * Sets a local block in the wire to block new requests for data to be responded.
+   * And sends a STOP event to the requesting peer (leecher).
+   */
   blockPeer(torrentInfoHash: string, wire: PaidStreamingWire) {
     wire.paidStreamingExtension.stop();
+    const {peerAccount, seedingChannelId} = wire.paidStreamingExtension;
+    const torrentPeers = this.peersList[torrentInfoHash];
+
     this.emit(ClientEvents.PEER_STATUS_CHANGED, {
-      torrentPeers: this.peersList[torrentInfoHash],
+      torrentPeers,
       torrentInfoHash,
-      peerAccount: wire.paidStreamingExtension.peerAccount,
-      seedingChannelId: wire.paidStreamingExtension.seedingChannelId
+      peerAccount,
+      seedingChannelId
     });
-    log.info(
-      {
-        from: this.peersList[torrentInfoHash],
-        peerAccount: wire.paidStreamingExtension.peerAccount,
-        seedingChannelId: wire.paidStreamingExtension.seedingChannelId
-      },
-      `<< blockPeer`
-    );
+    log.info({from: torrentPeers, peerAccount, seedingChannelId}, `<< blockPeer`);
   }
 
+  /**
+   * Removes a local block in the wire to block new requests for data to be responded.
+   * Triggers a re-processing of the cached-blocked-requests from the blocked peer.
+   * Sends a START event to the requesting peer (leecher).
+   */
   unblockPeer(torrentInfoHash: string, wire: PaidStreamingWire) {
+    const {peerAccount, seedingChannelId} = wire.paidStreamingExtension;
+    const torrentPeers = this.peersList[torrentInfoHash];
+
     // TODO: only unblock if the buffer is large enough
-    log.info(
-      {
-        from: Object.keys(this.peersList),
-        peerAccount: wire.paidStreamingExtension.peerAccount,
-        seedingChannelId: wire.paidStreamingExtension.seedingChannelId
-      },
-      '<< unblockPeer: start'
-    );
+    log.info({from: torrentPeers, peerAccount, seedingChannelId}, '<< unblockPeer: start');
     wire.paidStreamingExtension.start();
     this.emit(ClientEvents.PEER_STATUS_CHANGED, {
-      torrentPeers: this.peersList[torrentInfoHash],
+      torrentPeers,
       torrentInfoHash,
-      peerAccount: wire.paidStreamingExtension.peerAccount,
-      seedingChannelId: wire.paidStreamingExtension.seedingChannelId
+      peerAccount,
+      seedingChannelId
     });
-    log.info(
-      {
-        from: this.peersList[torrentInfoHash],
-        peerAccount: wire.paidStreamingExtension.peerAccount,
-        seedingChannelId: wire.paidStreamingExtension.seedingChannelId
-      },
-      '<< unblockPeer: finish'
-    );
+    log.info({from: torrentPeers, peerAccount, seedingChannelId}, '<< unblockPeer: finish');
   }
 
-  // Note, this is only used by unit tests.
+  /** Toggle between Blocked/Unlocked a channel for a torrent. */
```

Comment:
Should we move this to a test utility module since this is only used by tests?

Reply:
good idea!. Will do

Reply:
https://github.com/statechannels/monorepo/pull/2002/commits/8cd21365370688b097a10b0cec78efd544f801e6